lutris/util/script_thread.py

```diff
@@ -0,0 +1,45 @@
+import os
+import subprocess
+from threading import Thread
+
+from lutris.util.log import logger
+from lutris.gui import dialogs
+
+
+class ScriptThread():
+    def __init__(self, script):
+        self.script = script
+        self.script_thread = None
+
+    def call_process_with_err(self, arg):
+        """Handle subprocess errors in the new thread"""
+
+        try:
+            return subprocess.call(arg)
+
+        except PermissionError:
+            error = "Unable to execute " + arg + ", script is not executable"
+            logger.error(error)
+            dialogs.ErrorDialog(error)
+        except OSError:
+            error = "Unable to execute " + arg + ", script maybe missing Shebang"
```

Comment:
String interpolation

Reply:
I think its resolved ( I googled this and still didn't find a good explanation)